### ISSUE TITLE
Nuxt route priority

### ISSUE THREAD

**Issue by GautierDele** [m2sd/laravel-nuxt#4]
Hello,
It seems like `Nuxt::route()` is taking priority on some routes. I had to ignore api via:
```php
Nuxt::route('{path?}')->name('nuxt')->where('path', '^(?!nova|api.*$).*');
```

I am also using passport and it seems like not having trouble. Seems strange because route:list is puting the route generated at the end of the stack.
Could be interesting to also fix the issue forcing to ignore nova 😄

**Comment by jheslop25** [comment]
@GautierDele I'm having more or less the same issue. Honestly I'm profoundly regretting using this package. I'm using Twill.io as an admin/cms package, and its great, but when I changed the nuxt prefix to '/' from '/app' and tried to ignore api routes because of the issue you noted, it bricked my entire admin panel. even when I've reversed the changes I made I can't get it back. This is a clever idea, but its clearly not ready to be used.
